src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/index.jsx

```diff
@@ -15,12 +16,38 @@ import styled from 'react-emotion';
 import space from 'app/styles/space';
 import {withTheme} from 'emotion-theming';
 
+const api = new Client();
+
 class SentryApplicationRow extends React.PureComponent {
   static propTypes = {
     app: PropTypes.object.isRequired,
     orgId: PropTypes.string.isRequired,
   };
 
+  redirectUrl = install => {
+    const {orgId, app} = this.props;
+
+    return app.redirectUrl
+      ? `${app.redirectUrl}?installationId=${install.uuid}&code=${install.code}`
```

Comment:
Can redirectUrls contain query strings?

Reply:
We don't do any validation currently, but I was thinking we would eventually require these to just be simple URIs with no params. Maybe it's better to just support both and make this more query param tolerant, though. 🤔